app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -121,7 +121,21 @@ Fae.form.ajax = {
 
         // remotipart returns data inside textarea, let's grab it from there
         if ($html.is('textarea')) {
-          $html = $( $html.val() );
+          if ( $html.length > 1 ) {
+            var $textarea;
+
+            for (var i = 0; i < $html.length; i++ ) {
+              if ( $($html[i]).is('textarea') ) {
```

Comment:
Interesting, did not know that. So then what if L128 returns another Array?

Other notes: if `$textarea` is not reassigned in this loop, L135 will throw an error (`.text()`) and there should be a changelog item added for this.